.gitlab-ci.yml

```diff
@@ -49,10 +49,25 @@ build:
     paths:
       - image.tar
 
+build:testing:
+  stage: build
+  dependencies:
+    - build_prep
+  services:
+    - docker:dind
+  script:
+    - docker build -t $DOCKER_REPOSITORY:testing -f backend-acceptance-testing/Dockerfile.backend-tests backend-acceptance-testing
+    - docker save $DOCKER_REPOSITORY:testing > testingImage.tar
```

Comment:
Here also:

```suggestion
    - docker save $DOCKER_REPOSITORY:acceptance-testing > testingImage.tar
```

Reply:
argh, sorry for the oversights... should be aligned now!